vault/dendron.dev.setup.md

```diff
@@ -35,22 +35,31 @@ npm install
 
 - we recommend you use `vscode` to develop for dendron. there is a `dendron.code-workspace` file in the root of the monorepo that you should use when developing
 
-### Advanced
+### 4. Start Debugging
 
-You shouldn't need to use this in case something goes wrong with a build step or you want to save time by not running everything, `init.sh` is just a thin wrapper around the following scripts, each of which can be run individually
+To start an instance of the Dendron with the Debugger, Run `Extension (plugin-core)` from the debug panel in vscode
+
+![debugger](assets/images/start_debugger.gif)
```

Comment:
when publishing, the `assets/images` doesn't resolve properly but `/assets/images` does, both locally and remotely. could you a `/`?

Reply:
Sorry, missed this. Fixed!

And I see some re-org has already happened. Happy to re-open against `master`.

Reply:
thanks for fixing. re-open against master would be great. will leave this page alone in the meantime :)